package.json

```diff
@@ -1,23 +1,33 @@
 {
-  "name": "tgh-website",
+  "name": "taxmaps",
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@google-cloud/storage": "^4.7.0",
+    "@google-cloud/vision": "^1.11.0",
     "@testing-library/jest-dom": "^4.2.4",
-    "@testing-library/react": "^9.3.2",
-    "@testing-library/user-event": "^7.1.2",
+    "@testing-library/react": "^9.5.0",
+    "@testing-library/user-event": "^7.2.1",
+    "boostrap": "^2.0.0",
```

Comment:
What is this? I could only find this deprecated library https://libraries.io/npm/boostrap